pegasus/sites.v3/code.org/public/maker/index.haml

```diff
@@ -35,6 +35,7 @@ theme: responsive_full_width
 
 = view :section_divider_line
 
+%a{id: "explore_devices_link"}
```

Comment:
@kelbyhawn - just wanted to check with you on this change. I added this so that it could be linked from https://studio.code.org/maker/setup. Thanks!

Reply:
Thanks for tagging me! Instead of adding this new `%a` element you can add the id to the `%section` element below this on line 39.
```suggestion
%section#explore_devices_link
```

Reply:
Thank you! I updated accordingly.